app/assets/javascripts/discourse/app/static/prosemirror/core/keymap.js

```diff
@@ -26,10 +30,25 @@ export function buildKeymap(
 
   keys["Mod-z"] = undo;
   keys["Shift-Mod-z"] = redo;
-  keys["Backspace"] = undoInputRule;
+
+  const backspaceUnset = (state, dispatch, view) => {
+    const $pos = atBlockStart(state, view);
+    if (BACKSPACE_UNSET_NODES.includes($pos?.parent.type.name)) {
```

Comment:
```suggestion
    const pos = atBlockStart(state, view);
    if (BACKSPACE_UNSET_NODES.includes(pos?.parent.type.name)) {
```
this is extremely nitpicky, but I'm not sure if there is a need for this $, its not a convention we usually use from what I know? Happy to be wrong here too

Reply:
It is a convention from ProseMirror when we have a `ResolvedPos` instance. For example, https://prosemirror.net/docs/ref/#state.Selection has both from/to and $from/$to, the first is a numeric position and the second is a `ResolvedPos` instance.

Reply:
Ok that's fair :+1: